credit/README.md

```diff
@@ -12,29 +12,29 @@ So what's the secret formula?  Well, most cards use an algorithm invented by Han
 1. Add the sum to the sum of the digits that weren't multiplied by 2.
 1. If the total's last digit is 0 (or, put more formally, if the total modulo 10 is congruent to 0), the number is valid!
 
-That's kind of confusing, so let's try an example with David's AmEx: 378282246310005.
+That's kind of confusing, so let's try an example with David's Visa: 4003600000000014.
 
 1. For the sake of discussion, let's first underline every other digit, starting with the number's second-to-last digit:
    
-   3<u>7</u>8<u>2</u>8<u>2</u>2<u>4</u>6<u>3</u>1<u>0</u>0<u>0</u>5
+   <u>4</u>0<u>0</u>3<u>6</u>0<u>0</u>0<u>0</u>0<u>0</u>0<u>0</u>0<u>1</u>4
 
    Okay, let's multiply each of the underlined digits by 2:
 
-   7•2 + 2•2 + 2•2 + 4•2 + 3•2 + 0•2 + 0•2
+   4•2 + 0•2 + 6•2 + 0•2 + 0•2 + 0•2 + 0•2 + 1•2
```

Comment:
@dmalan @curiouskiwi it doesn't really matter but just for consistency, should we start here with the second to last digit?

Reply:
The original version had them left to right, so I left it that way.  But showing from last to first might be even clearer. I'll update.